### PR TITLE
Upgrade to ansible version 1.9.3-rc1-edx.

### PR DESCRIPTION
**Pull request by smarnach**
JIRA ticket: [OLIVE-18](https://openedx.atlassian.net/browse/OLIVE-18)

This is a replacement for #19, which got closed and can't be re-opened. See also edx/edx-analytics-pipeline#209

@mulby Can you please take a look?

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @smarnach! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-analytics-configuration&number=27

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ansible==1.4.4	# GPL v3
+ansible==1.9.4  # GPL v3
```
Comment (e0d) [inline requirements.txt]: If the plan if to fold analytics configuration back into configuration I think we should use our fork of ansible, ```git+https://github.com/edx/ansible.git@stable-1.9.3-rc1-edx#egg=ansible==1.9.3-edx```

There are some type conversion bugs fixed there.

@feanil can you confirm whether or not those changes have been accepted upstream?

**Comment by e0d** [comment]
After small comment, :+1:

**Comment by mulby** [comment]
:+1:

**Comment by smarnach** [comment]
@e0d @mulby Thanks!  I'll wait for @feanil to comment before merging.

**Comment by smarnach** [comment]
@e0d @mulby I just noticed I don't have write permission for this repo, so one of you will have to merge this.

File: requirements.txt (hunk 1 above)
Comment (feanil) [inline requirements.txt]: All changes have been accepted upstream but not all of the changes will be available before ansible 2.0  I recommend using the fork @e0d mentioned unless moving to ansible 2.0 is an option.

**Comment by smarnach** [comment]
@feanil Thanks for confirming.  I've changed the version to the edx fork of 1.9.3 now, since I'm not sure this would work with 2.x, and it probably makes sense to use the same version as in the configuration repo anyway.

@e0d @mulby Could one of you please merge the PR?

**Comment by mulby** [comment]
How critical is this change? It will require us to update all of our jenkins jobs I think...

I encountered 2 major issues:

1. Ansible wouldn't install using distribute, I had to go into the advanced settings for the virtualenv builder in jenkins and switch it to "setuptools". I would have to do this on *all* of our jobs.
2. When I try to run with our standard EXTRA_VARS I get the following error message:
"ERROR: Expecting property name enclosed in double quotes: line 2 column 3 (char 4)"

**Comment by smarnach** [comment]
@mulby That's unfortunate.  We've been using the updated version for a while and didn't run into any problems.  The reason for this upgrade were problem with the old verison @mtyaka experienced, see https://github.com/edx/edx-analytics-pipeline/pull/209#issuecomment-191629410.  Matjaz, do you have any additional comments?

**Comment by mtyaka** [comment]
It would make sense to revisit this after we get jenkins setup automated.

**Pull request merged**